city_scrapers/spiders/phipa_trb.py

```diff
@@ -0,0 +1,123 @@
+import json
+from datetime import datetime, timedelta
+
+import pytz
+from city_scrapers_core.constants import BOARD, CANCELLED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy import Request
+
+
+class PhipaTrbSpider(CityScrapersSpider):
+    name = "phipa_trb"
+    agency = "Philadelphia Tax Review Board"
+    timezone = "America/New_York"
+    calendar_id = "taxreviewboard@gmail.com"
+    calendar_website = (
+        f"https://calendar.google.com/calendar/u/0/embed?src={calendar_id}"
+    )
+    location = {
+        "name": "Land Title Building",
+        "address": "100 S. Broad Street - Suite 400, Philadelphia, Pennsylvania 19110-1099",  # noqa
+    }
+    agendas_link = {
+        "title": "Agenda page",
+        "href": "https://www.phila.gov/documents/tax-review-board-agendas/",
+    }
+
+    def start_requests(self):
+        api_key = self.settings.get("GOOGLE_CLOUD_API_KEY")
+        if not api_key:
+            raise ValueError("No GOOGLE_CLOUD_API_KEY provided")
+
+        # calculate the date two months ago
+        current_datetime = datetime.utcnow()
+        two_months_prior = current_datetime - timedelta(days=60)
+        minTimeVal = two_months_prior.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+        # Construct the URL with query parameters
+        url = f"https://www.googleapis.com/calendar/v3/calendars/{self.calendar_id}/events?key={api_key}&maxResults=500&timeMin={minTimeVal}"  # noqa
+        yield Request(url, self.parse)
+
+    def parse(self, response):
+        """
+        Parse the response from the Google Calendar API and yield Meeting items.
+        """
+        data = json.loads(response.text)
+        for item in data["items"]:
+            all_day = True if "date" in item["start"] else False
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=BOARD,
+                start=self._parse_datetime(item["start"]),
+                end=self._parse_datetime(item["end"]),
+                all_day=all_day,
+                time_notes=None,
+                location=self.location,
+                links=self._parse_links(item),
+                source=self.calendar_website,
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse meeting title."""
+        return item.get("summary") or ""
+
+    def _parse_description(self, item):
+        """Parse meeting description. Description text for this
+        agency's Google Calendar tends to contain HTML so we convert
+        it to plain text."""
+        if "description" not in item:
+            return ""
+        return item.get("description") or ""
```

Comment:
Agreed. Sorry about that. I copied over a lot of of code from our other Google Calendar spider (#14) and it looks like this method should have been cleaned up. I've improved the docstring as well, which wasn't accurate, since it referred to HTML parsing that was relevant to the other spider but not this one.